lib/phoenix_inline_svg/helpers.ex

```diff
@@ -264,17 +219,34 @@ defmodule PhoenixInlineSvg.Helpers do
     |> Enum.into([])
   end
 
-  defp create_cached_svg_image({collection, name}, svgs_path) do
-    filename = name |> String.split(".") |> List.first
+  defp create_cached_svg_image({collection, name}) do
+    filename =
+      name
+      |> String.split("/")
```

Comment:
Good catch. I was not thinking of this case since this package has the "collections" concept.

This would currently work with case of `svg_image("icon",  "some")`.

I personally like the API for `svg_image("some/icon")` instead of the "collections" API and would move to deprecate "collections", but would call that out of scope for this PR.

I'll see if I can get this working for now.